src/dynamicprompts/utils.py

```diff
@@ -97,3 +97,8 @@ def choose_without_replacement(
             weights.remove(weights[values.index(chosen)])
             values.remove(chosen)
         return chosen_values
+
+
+def _fix_max_bound(max_bound: int | None, values: any | int) -> int:
```

Comment:
`any` is the built-in `any()` function; you'd want `typing.Any`.
```suggestion
def _fix_max_bound(max_bound: int | None, values: Any | int) -> int:
```

However, looking at the code, the other option isn't `Any`; it's `collections.abc.Sized` (something that can be `len()`'d).

```suggestion
def _fix_max_bound(max_bound: int | None, values: Sized | int) -> int:
```